packages/publish/src/publish.ts

```diff
@@ -11,11 +11,6 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
-
-// define types for a few modules used by probot that do not have their
-// own definitions published. Before taking this step, folks should first
-// check whether type bindings are already published.
```

Comment:
Did you intend to remove this note?

Reply:
Yeah - it didn't look to apply to anything actually happening below.  I don't think it was relevant.